Corona-Warn-App/src/main/java/de/rki/coronawarnapp/presencetracing/checkins/qrcode/QrCodeGenerator.kt

```diff
@@ -38,7 +38,8 @@ class QrCodeGenerator @Inject constructor(
 
         val hints = mapOf(
             EncodeHintType.ERROR_CORRECTION to correctionLevel,
-            EncodeHintType.MARGIN to margin
+            EncodeHintType.MARGIN to margin,
+            EncodeHintType.CHARACTER_SET to Charsets.UTF_8
```

Comment:
this would cause a crash on older Android versions, did you test it on Android 6 ?
I had it before and remove it on purpose. I would suggest to add SDK version check and or make it configurable 
as this also affecting Presence Tracing Qr-Codes

Reply:
No i did not, i do not know if i version check would help here, qr codes litterally cannot be parsed by the ui in a different encoding, is there a specific reason for the crash that we could work around?

Reply:
should be fixed by the latest commit, turned out to just be a issue with .toString evaluation results